apps/condo/domains/organization/hooks/useOrganizationNameValidator.tsx

```diff
@@ -0,0 +1,18 @@
+import { Rule } from 'rc-field-form/lib/interface'
+import { useIntl } from 'react-intl'
+import { isValidName } from '../utils/name.utils'
+
+export const useOrganizationNameValidator = (): Rule => {
```

Comment:
Move this code to the place of usage. There in non reusable hook

Reply:
done